package.json

```
@@ -32,28 +32,28 @@
   },
   "homepage": "https://github.com/graphile/postgis#readme",
   "peerDependencies": {
-    "graphile-build": "^4.3.0",
-    "graphile-build-pg": "^4.3.0",
+    "graphile-build": "^4.4.4",
+    "graphile-build-pg": "^4.4.4",
     "graphql": ">=0.6 <15",
-    "pg-sql2": "^2.2.1"
+    "pg-sql2": "^4.4.3"
```

The change to `graphile-build` and `graphile-build-pg` is a breaking change as it says to use the postgis plugin you must be running the latest PostGraphile (where that wasn't previously the case). It's safe to update the `yarn.lock` (which says which version we're _actually_ using) and the `devDependencies` but `peerDependencies` and `dependencies` must be treated more carefully. For `pg-sql2` we want to explicitly state that we support the full range, so:

```suggestion
    "pg-sql2": ">=2.2.1 <5"
```

---

(For graphile-build and graphile-build-pg we already state we support 4.4.4 since ^4.3.0 includes 4.4.0, so we can just change those back to the old values.)

---

Fixed!